src/views/settings/Security/Session.vue

```diff
@@ -1,35 +1,88 @@
   </IBox>
 </template>
 
 <script>
 import IBox from '@/components/Common/IBox/index.vue'
 import GenericCreateUpdateForm from '@/layout/components/GenericCreateUpdateForm/index.vue'
+import WatermarkHelpDialog from './WatermarkHelpDialog.vue'
 
 export default {
   name: 'SessionSecurity',
-  components: { GenericCreateUpdateForm, IBox },
+  components: { GenericCreateUpdateForm, IBox, WatermarkHelpDialog },
   data() {
     return {
+      showHelpDialog: false,
       config: {
         url: '/api/v1/settings/setting/?category=security_session',
         hasDetailInMsg: false,
         fields: [
           [
             this.$t('Basic'),
             [
-              'SECURITY_WATERMARK_ENABLED',
               'SECURITY_SESSION_SHARE',
               'SESSION_EXPIRE_AT_BROWSER_CLOSE',
               'VIEW_ASSET_ONLINE_SESSION_INFO',
               'SECURITY_MAX_IDLE_TIME',
               'SECURITY_MAX_SESSION_TIME'
             ]
+          ],
+          [
+            this.$t('Watermark'),
+            this.$store.getters.hasValidLicense ? [
+              'SECURITY_WATERMARK_ENABLED',
+              'SECURITY_WATERMARK_SESSION_CONTENT',
+              'SECURITY_WATERMARK_CONSOLE_CONTENT',
+              'SECURITY_WATERMARK_COLOR',
+              'SECURITY_WATERMARK_FONT_SIZE',
+              'SECURITY_WATERMARK_HEIGHT',
+              'SECURITY_WATERMARK_WIDTH',
+              'SECURITY_WATERMARK_ROTATE'
+            ] : []
           ]
         ],
-        fieldsMeta: {},
+        fieldsMeta: {
+          SECURITY_WATERMARK_SESSION_CONTENT: {
+            helpTextFormatter: () => {
+              const handleClick = () => {
+                this.showHelpDialog = true
+              }
+              return (
+                <i onClick={handleClick} class='fa fa-external-link'>{this.$t('Help')}</i>
+              )
+            }
+          },
+          SECURITY_WATERMARK_CONSOLE_CONTENT: {
+            helpTextFormatter: () => {
+              const handleClick = () => {
+                this.showHelpDialog = true
+              }
+              return (
+                <i onClick={handleClick} class='fa fa-external-link'>{this.$t('Help')}</i>
+              )
+            }
+          },
+          SECURITY_WATERMARK_COLOR: {
+            component: 'el-color-picker',
+            el: {
+              size: 'small',
+              'show-alpha': true,
+              predefine: [
+                'rgba(128, 128, 128, 0.25)',
+                'rgba(0, 0, 0, 0.15)',
+                'rgba(255, 255, 255, 0.2)',
+                'rgba(176, 176, 176, 0.3)',
+                'rgba(91, 155, 213, 0.25)',
+                'rgba(192, 80, 77, 0.2)',
+                'rgba(112, 173, 71, 0.2)',
+                'rgba(255, 192, 0, 0.18)'
+              ]
+            }
+          }
+        },
         onSubmit: async(validValues) => {
           const url = '/api/v1/settings/setting/?category=security_session'
 
```

Comment:
The code is a template file that defines a custom dialog inside an `IBox` component within the Vue framework. Here are some differences:

- The original version includes a reference to WatermarkHelpDialog from the same folder named WatermarkHelpDialog.vue which was not included here.

Here's how we could optimize this based on these comments while keeping it concise:
1. Remove duplicate references.
2. Add missing components like IBox and watermarkhelpdialog if needed.